packages/vue/src/components/basic/ReactiveComponent.jsx

```diff
@@ -89,17 +122,17 @@ const ReactiveComponent = {
 	watch: {
 		hits(newVal, oldVal) {
 			if (!isEqual(newVal, oldVal)) {
-				this.$emit('allData', parseHits(newVal), oldVal);
+				this.$emit('data', parseHits(newVal), oldVal);
 			}
 		},
 		aggregations(newVal, oldVal) {
 			if (!isEqual(newVal, oldVal)) {
-				this.$emit('allData', parseHits(newVal), oldVal);
+				this.$emit('data', parseHits(newVal), oldVal);
 			}
 		},
 		aggregationData(newVal, oldVal) {
 			if (!isEqual(newVal, oldVal)) {
-				this.$emit('allData', newVal, oldVal);
+				this.$emit('data', newVal, oldVal);
```

Comment:
@ShahAnuj2610 It should be `this.$emit('data', this.getData())`.
Check to react implementation for `onData`.